pkg/daemon/ceph/ganesha/daemon.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ganesha
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rook/rook/pkg/clusterd"
+)
+
+func Run(context *clusterd.Context, name string) error {
+
+	// create the run dir for dbus to start
+	if err := os.Mkdir("/run/dbus", 0755); err != nil {
+		logger.Errorf("Couldn't create /run/dbus: %+v", err)
+	}
+
+	logger.Infof("Starting dbus daemon")
+	if err := context.Executor.ExecuteCommand(false, "", "/usr/bin/dbus-daemon", "--system", "--nopidfile"); err != nil {
```

Comment:
I'll go with `dbus-daemon` only since we don't if all the distros will have the same path for that daemon.

Reply:
Makes sense, but we are assuming that it'll be in $PATH that way.

Reply:
We are assuming that for a bunch of other commands already, see `ganesha.nfsd` below.

Reply:
Good point.